docs/guides/publishing/playground.md

```diff
@@ -237,10 +237,10 @@ You can optionally render embedded notebooks in read-only mode by appending
 ### Embedding from code
 
 You can also embed marimo notebook from its string representation (i.e.,
-the notebook file's code), with the URL
+the notebook file's code), using the `code` query parameter:
 
 ```
-https://marimo.app?embed=true&show-chrome=false#code/<encoded-uri-component>
+https://marimo.app?embed=true&show-chrome=false&code=<encoded-uri-component>
```

Comment:
this has a much smaller URL limit. was this not working before?

Reply:
documented using the url hash with lz compression